recipes-kernel/linux/linux-mainline/lima.conf

```diff
@@ -1,3 +0,0 @@
-# lima kernel module configuration
-softdep lima pre: sun4i-drm sun4i-drm-hdmi sun4i-frontend sun4i-tcon sun4i-backend
-options lima job_hang_limit=10
```

Comment:
The patch was deleted because it was incorporated into the mainline linux. This configuration is now understood by the kernel without the special patch. This is still an important piece of configuration. Without it the device hangs occasionally.

Reply:
The presence of this file causes a build breakage, because it is not included in any binary package. So either delete it (what this commit does), or add it to the FILES variable of a package.

Reply:
After https://github.com/Openvario/meta-openvario/pull/188/commits/dbfcb6e1286c24435befaddd45bfdf8e609f2735 the module parameter must be moved to the kernel command line.

Reply:
Dropped for now (which keeps the build breakages), and I'll later figure out how to add this to the kernel command line.

Reply:
For the record, the problem is documented #71, the epic debugging effort.